src/magento-payments/payouts.md

```diff
@@ -13,42 +13,36 @@ You can download payout transactions in a .csv file format for use in existing a
 
 ## Availability
 
-On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts** to see payout transactions.
+On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts**.
 
 ![Magento Admin - Payouts]({% link images/images-ee/system-reports.png %}){: .zoom}
 _Payout transactions in the Admin_
 
-From the Payouts view, you can customize the timeframe of the payouts you want to view by selecting specific dates or viewing transactions for pre-set periods, such as _Last 30 days_ or _Year to date_.
-
 ## View transactions
 
 By default, 30 days of transactions are shown in the grid.
 
 Scroll to the left and right to view [information for each payout transaction](#column-descriptions) in the daily report, including transaction date, reference ID, invoice number, and payment method details.
 
-The number of payout transactions that are visible, for the default 30 day timeframe or a [customized timeframe](#customize-transactions-timeframe), are shown above the payouts grid as a records count.
-
 ### Customize transactions timeframe
 
-If you want to see all payout transactions for a time period other than the default last 30 days, you can customize the payout transaction timeframe for specified dates.
+From the Payouts view, you can customize the timeframe for the payout transactions you want to view by entering a specific timeframe or selecting a date range from the date picker:
```

Comment:
I am not fond of 'timeframe'. 
We can either enter a date range through the input fields or select a date range from the calendar date picker.
Maybe:
From the Payouts view, you can customize the date range for payout transactions you want to view by entering specific dates or selecting a range from the date picker.